src/deluge/gui/l10n/strings.h

```diff
@@ -676,17 +676,10 @@ enum class String : size_t {
 	STRING_FOR_MASTER_PAN,
 	STRING_FOR_SIDECHAIN_LEVEL,
 
```

Comment:
re-grouped automation view strings together at the bottom to make it them easier to find and adjust if necessary.